src/app/image-group-page/image-group-page.component.pug

```diff
@@ -4,12 +4,8 @@ div
     .row.no-gutters.row--search-bar
       .col-md-5
         ang-search
-    .row
-      .col-12.pb-4
-        h3#pageType Browse Group
-        .row
-          h1#pageTitle.h1--title-block.col-md-9 {{ ig.name }}
-          
+    ang-group-title([ig]="ig")
```

Comment:
We should probably ngIf this to only load once the ig is loaded. Right now the half loaded header is a little rough